tools/genv8constants.py

```diff
@@ -84,10 +84,10 @@ def out_define():
       # 6-character margin, 2-characters + 1 space for each field
       idx = 6 + i * 3;
       octetstr = line[idx:idx+3]
-      if not numpattern.match(octetstr):
+      if curr_octet > octets:
```

Comment:
Why did you change the order of these two?

Reply:
I change because order change not affect logic, and number comparison faster then regex.